### ISSUE TITLE
CHANGELOG verb tenses are inconsistent

### ISSUE THREAD

**Issue by GitMje** [gaelcolas/Sampler#226]
The CHANGELOG has a mix of present, past, and imperative tenses for the first verb ... e.g., Update, Updating, and Updated can be seen.  As I'm sure that you're aware, [git suggests using imperative for commit messages](https://www.danclarke.com/git-tense).  I agree with the author of the blog post that it really doesn't matter as long as it's consistently followed.  It would be helpful to add some directions to the CHANGELOG to tell people which tense to use.

**Comment by johlju** [comment]
I agree it would be great if it is consistent. Not sure the imperative for commit messages are good for the change log since it is also used for the release notes.
For commit messages I like the commit messages to be imperative because the say 'this diff do this to the source/repo/pipeline' and it is contributors that reads this. The entries in the change log are different, they don't reflect each commit/diff but rather what changed from a user perspective and they should (IMHO) be descriptive what the change does and mean to the user. The change log is also used as the release notes when the modules is published. Release notes can be seen under Package Details here https://www.powershellgallery.com/packages/Sampler/ and is taken from the module manifest.

IMHO using imperative tense is great for commit messages, not for the change log, but can be because I'm not used to it. 🤔

**Comment by johlju** [comment]
I let @gaelcolas give his thoughts in this.

**Comment by GitMje** [comment]
@johlju I agree that I prefer imperative tense for commit messages and past tense for change log.

**Comment by gaelcolas** [comment]
> it really doesn't matter as long as it's consistently followed

Agreed. We ought to do a better job at this.

> add some directions to the CHANGELOG to tell people which tense to use

At least in the template, yes. And the documentation (seriously lacking here, I know).
But yeah, we could try to do more.

Then, like most of the guidelines and style, if we don't have an automatic way to enforce it, it's difficult to really be consistent. There's definitely room for improvement, and we could already do a better job with cspell.

If you have some ideas please let us know, and yeah, we would probably accepts PR going in that direction.
